Reason: Set RUN_NEON for cross compiling with arm architectures

diff -ru CMakeLists.txt CMakeLists.txt
--- CMakeLists.txt	2020-10-28 01:45:30.000000000 +0100
+++ d/CMakeLists.txt	2020-12-29 09:49:49.006446795 +0100
@@ -8,6 +8,7 @@
 set(direwolf_VERSION_PATCH "0")
 set(direwolf_VERSION_SUFFIX "")
 
+set(RUN_NEON "advanced")
 # options
 option(FORCE_SSE "Compile with SSE instruction only" OFF)
 option(FORCE_SSSE3 "Compile with SSSE3 instruction only" OFF)
